src/components/Trends/TrendsTable/TrendsTable.js

```diff
@@ -21,116 +24,134 @@ const columns = [
     Header: '#',
     accessor: 'index',
     width: 35,
-    headerClassName: styles.headerIndex
+    headerClassName: styles.headerIndex,
   },
   {
     Header: 'Word',
-    accessor: 'word'
+    accessor: 'word',
   },
   {
-    Header: 'Hyped score',
-    accessor: 'score'
+    Header: 'Hype score',
+    accessor: 'score',
   },
   {
     Header: 'Social volume',
-    accessor: 'volume'
-  }
+    accessor: 'volume',
+  },
 ]
 
-const NumberCircle = props => (
-  <div {...props} className={styles.insights__number} />
+const NumberCircle = ({ className, ...props }) => (
+  <div {...props} className={cx(className, styles.insights__number)} />
+)
+
+const Explanation = ({ text, children, ...props }) => (
+  <Tooltip
+    offsetY={5}
+    {...props}
+    className={styles.explanation}
+    closeTimeout={0}
+    trigger={children}
+  >
+    {text}
+  </Tooltip>
 )
 
+const getTrGroupProps = (_, rowInfo) => {
+  return {
+    onClick: ({ target, currentTarget }) => {
+      let node = target
+      while (node && node !== currentTarget) {
+        if (
+          node.classList.contains(styles.action) ||
+          node.classList.contains(styles.checkbox)
+        ) {
+          return
+        }
+        node = node.parentNode
+      }
+      store.dispatch(push(`/labs/trends/explore/${rowInfo.original.rawWord}`))
```

Comment:
@DmitriVanGuard  We loose here "a" ability to open this in new tab. And spiders lose these links.

Reply:
No, we don't and [here is why](https://github.com/santiment/app/blob/2aa1a1174a2cd06c5dd7e0add1b388c89a323865/src/components/Trends/TrendsTable/TrendsTable.js#L246)